src/languages/sqf.js

```diff
@@ -2156,12 +2278,13 @@ export default function(hljs) {
     'shownCompass',
     'shownCuratorCompass',
     'showNewEditorObject',
-    'shownGPS',
+    'shownGps',
```

Comment:
```suggestion
    'shownGPS',
```
https://community.bistudio.com/wiki/shownGPS

Reply:
Is the language perhaps case insensitive?  That would explain a lot.